vms/authentication/tests/test_services.py

```diff
@@ -1,3 +1,19 @@
 from django.test import TestCase
+from authentication.views import login_process
+from django.contrib.auth.models import User
+from django.test.client import RequestFactory
+
 
 # Create your tests here.
+class TestLoginProcess(TestCase):
+
+	def testLoginViewRateLimit(self):
+		request = RequestFactory().post("/login")
+		request.user = User.objects.create_user(
+			username='Test User', email='test@a.com', password='Password')
+		for loginAttemp in range(25):
```

Comment:
@rituranjna I think you should try changing this. Which redirection are you referring to?